app/src/main/java/com/framgia/englishconversation/screen/createPost/CreatePostViewModel.java

```diff
@@ -483,78 +473,67 @@ private void selectImage() {
+        releaseExoAudioPlayer();
         mTimelineModel.getMedias().remove(mediaModel);
         mMediaAdapter.removeItem(mediaModel);
     }
 
-    private void initMedia(String filePath) {
-        mMediaPlayer = new MediaPlayer();
-        try {
-            mMediaPlayer.setDataSource(filePath);
-            mMediaPlayer.prepare();
-        } catch (IOException e) {
-            mNavigator.showToast(mActivity.getString(R.string.error_init_media));
-        }
-    }
-
     private void showAudioDialog() {
         if (mActivity.getExternalCacheDir() == null) {
             return;
         }
         String fileName = mActivity.getString(R.string.prefix_audio_file_name)
-            + System.currentTimeMillis()
-            + Constant.DEFAULT_FORMAT_AUDIO;
+                + System.currentTimeMillis()
+                + Constant.DEFAULT_FORMAT_AUDIO;
         mAudioFilePath = mActivity.getExternalCacheDir().getAbsolutePath() + "/" + fileName;
         RecordingAudioBuilder.with(mActivity, mRecordingAudioDialog)
-            .setFileName(fileName)
-            .setFilePath(mAudioFilePath)
-            .setAudioSource(AudioSource.MIC)
-            .showRecordingAudioFromActivity();
+                .setFileName(fileName)
+                .setFilePath(mAudioFilePath)
+                .setAudioSource(AudioSource.MIC)
+                .showRecordingAudioFromActivity();
     }
 
     public void onAudioConversationClick(final ConversationModel conversations) {
         showAudioDialog();
         RecordingAudioDialog.OnRecordingAudioListener recordingAudioClickListener =
-            new RecordingAudioDialog.OnRecordingAudioListener() {
-                @Override
-                public void onRecordingAudioClick(String filePath, String fileName) {
-                    MediaModel record = new MediaModel(MediaModel.MediaType.AUDIO);
-                    record.setId(UUID.randomUUID().toString());
-                    record.setUrl(filePath);
-                    record.setName(fileName);
-                    initMedia(filePath);
-                    conversations.setMediaModel(record);
-                }
+                new RecordingAudioDialog.OnRecordingAudioListener() {
+                    @Override
+                    public void onRecordingAudioClick(String filePath, String fileName) {
+                        MediaModel record = new MediaModel(MediaModel.MediaType.AUDIO);
+                        record.setId(UUID.randomUUID().toString());
+                        record.setUrl(filePath);
+                        record.setName(fileName);
+                        conversations.setMediaModel(record);
+                    }
 
-                @Override
-                public void onRecordCancel() {
-                    // no ops
-                }
-            };
+                    @Override
+                    public void onRecordCancel() {
+                        // no ops
+                    }
+                };
```

Comment:
ERROR: 'object def rcurly' have incorrect indentation level 16, expected level should be one of the following: 8, 12.